package.json

```diff
@@ -2,36 +2,33 @@
   "name": "@mighty-justice/smart-bool",
   "version": "1.0.0",
   "description": "Eliminate your single-line boolean setting methods",
-  "main": "dist/SmartBool.cjs.js",
+  "main": "dist/index.js",
   "module": "dist/SmartBool.esm.js",
-  "browser": "dist/SmartBool.umd.js",
+  "typings": "dist/index.d.ts",
   "types": "dist/index.d.ts",
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
-    "build": "npm run build:clean && npm run build:types && npm run build:js",
-    "build:clean": "rm -r dist/* || true",
-    "build:js": "rollup -c",
-    "build:types": "tsc --emitDeclarationOnly",
+    "analyze": "source-map-explorer 'dist/*.js'",
+    "build": "tsdx build",
     "deploy": "np",
     "dev": "rollup -c -w",
-    "docs:build": "npm run docs:prepare && gitbook build",
-    "docs:deploy": "gh-pages -d _book",
-    "docs:prepare": "gitbook install",
-    "docs:watch": "npm run docs:prepare && gitbook serve",
-    "lint": "tslint --project .",
-    "postpublish": "npm run docs:build && npm run docs:deploy",
+    "format": "tsdx lint --fix",
+    "lint": "tsdx lint",
+    "prepare": "tsdx build",
     "preversion": "npm run lint",
-    "test": "jest",
-    "test:watch": "npm run test/*.js -- -w",
-    "version": "npm run build && git add dist",
-    "view-source-map": "source-map-explorer --html dist/index.js dist/index.map > source.html && open source.html"
+    "start": "tsdx watch",
+    "test": "tsdx test",
+    "version": "npm run build && git add dist"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/mighty-justice/smart-bool.git"
   },
+  "browserslist": [
```

Comment:
are we sure this isn't going to completely mess up our app at runtime? here's the list of available browsers for that setting:
```
and_chr 71
chrome 72
chrome 71
ios_saf 12.0-12.1
```

their default:
```
and_chr 71
and_ff 64
and_qq 1.2
and_uc 11.8
android 67
android 4.4.3-4.4.4
baidu 7.12
chrome 73
chrome 72
chrome 71
edge 18
edge 17
firefox 66
firefox 65
firefox 64
firefox 60
ie 11
ie_mob 11
ios_saf 12.0-12.1
ios_saf 11.3-11.4
op_mini all
op_mob 46
opera 58
opera 57
safari 12
safari 11.1
samsung 8.2
samsung 7.2-7.4
samsung 4
```